### ISSUE TITLE
[Bug] Accessibility Block apps. Guugler banned to use accessibility (Rethinkdns v5.5u)

### ISSUE THREAD

**Issue by cesarmarcanove** [celzero/rethink-app#2709]
Accessibility bug. Guugler has been banned to use accessibility in Rethink v5.5u, seems an "black ghost banning". Fix this at bug.

<img width="987" height="2227" alt="Image" src="https://github.com/user-attachments/assets/8848b7f5-0921-42b4-b789-8c75733da201" />
<img width="987" height="2227" alt="Image" src="https://github.com/user-attachments/assets/cbdc9e1e-924d-44bc-b996-07ad100b4422" />
<img width="987" height="2227" alt="Image" src="https://github.com/user-attachments/assets/f3a6a60c-6bfd-49fb-b77c-1e8b92802012" />
<img width="987" height="2227" alt="Image" src="https://github.com/user-attachments/assets/12e447c4-386e-46d9-9483-7f20e7b7fde2" />
